crates/bitwarden/src/client/encryption_settings.rs

```diff
@@ -104,9 +104,8 @@ impl EncryptionSettings {
         let (key, mac_key) = crate::crypto::stretch_key_password(
             password.as_bytes(),
             auth.email.as_bytes(),
-            auth.kdf_iterations,
-        )
-        .map_err(|_| CryptoError::KeyStretch)?;
```

Comment:
Is this error unused now?

Reply:
Yeah, seems like we have no other uses of that variant anymore, I'll remove it.